src/states/bootstrapping_peer.rs

```diff
@@ -41,52 +42,50 @@ pub struct BootstrappingPeer {
     bootstrap_connection: Option<(ConnectionInfo, u64)>,
     network_service: NetworkService,
     full_id: FullId,
-    min_section_size: usize,
     peer_map: PeerMap,
     timer: Timer,
     relocate_details: Option<SignedRelocateDetails>,
+    network_cfg: NetworkParams,
 }
 
 impl BootstrappingPeer {
-    /// Create `BootstrappingPeer` for a node that is joining the network (either for the first
-    /// time or rejoining).
     pub fn new(
         mut network_service: NetworkService,
         full_id: FullId,
-        min_section_size: usize,
+        network_cfg: NetworkParams,
         timer: Timer,
     ) -> Self {
         network_service.service_mut().bootstrap();
         Self {
             network_service,
             full_id,
-            min_section_size,
-            timer,
+            timer: timer,
```

Comment:
`timer: timer` is redundant. Just `timer` is enough.

Reply:
Good catch!